foreach-associatve-loop-function.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+function title($title) {
+    echo "<h2> " . $title . " </h2>";
+}
+
+
+function viewPerson($array) {
+    foreach ($array as $person) {
+        title('SuperHero: ');
```

Comment:
@Katrin12323 very good idea and use of functions. Keep the great work my sunshine girl :))) @Katrin12323 TOP^^ TOP^^